c/include/ml-api-common.h

```diff
@@ -404,6 +404,70 @@ const char * ml_error (void);
  */
 const char * ml_strerror (int errnum);
 
+/*************
+ * ML OPTION *
+ *************/
+
+/**
+ * @brief A handle of a ml-option instance.
+ * @since_tizen 7.0
+ */
+typedef void *ml_option_h;
+
+/**
+ * @brief The function to be called when destroying the data of machine learning API.
+ * @since_tizen 7.0
+ * @param[in,out] data The data to be passed to the callback function.
+ */
+//  * @return @c 0 on success. Otherwise a negative error value.
+typedef void (*ml_data_destroy_cb) (void *data);
+
+/**
+ * @brief Create ml-option instance.
+ * @details Some detail information should be written here.
```

Comment:
IMO, line 423, 427, and 440 are unnecessary.

Reply:
Right, I will update the docs soon